bulk-install-app/main.py

```diff
@@ -31,32 +30,30 @@
     for organization_tuple in organizations(session, uninstall=args.uninstall):
         # Handle installation for one organization
         target_name, target_url = organization_tuple
-        if not args.uninstall:
-            print(f'Found organization/user: {target_name}, to install app: {GITHUB_APP_NAME}.')
+        action = 'install' if not args.uninstall else 'uninstall'
+        print(f'Found organization/user: {target_name}, to {action} app: {GITHUB_APP_NAME}.')
 
+        if not args.uninstall:
             # Marking installation success
-            success = install_and_uninstall_if_necessary(
-                session=session, target_name=target_name, target_install_url=target_url
-            )
```

Comment:
Before we tried to transactionalize the installation, i.e. if the installation succeeded but redirect failed (would redirect to our APIs) we would attempt to uninstall the app and suggest a re-run of the script

Since we don't redirect to our APIs after installations / don't have callbacks anymore, we don't really need this - simplified to only do the installation